Casks/cube.rb

```diff
@@ -0,0 +1,12 @@
+cask 'cube' do
+  version '4.4'
+  sha256 '20698456ed2c19bf0bfb5a9e4ea74a761248e86a741c15d6f525ec8c5e20897c'
+
+  # apps.fz-juelich.de/scalasca/releases/cube was verified as official when first introduced to the cask
+  url "https://apps.fz-juelich.de/scalasca/releases/cube/#{version.major}/dist/Cube-Compressed.dmg"
```

Comment:
I think this needs to be changed to:

```suggestion
  url "https://apps.fz-juelich.de/scalasca/releases/cube/#{version}/dist/Cube-Compressed.dmg"
```

I'm going to make edits and force push (after testing locally)

Reply:
Ah, yes, my mistake. Don’t worry. Already fixed.

Also, never a need to force push when fixing a cask. We prefer you don’t, even, so changes are easier to track. We always `squash and merge`.

Reply:
I just was reading through the docs and noticed that after writing that comment.